.github/dependabot.yaml

```diff
@@ -1,9 +1,56 @@
 version: 2
 updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+      day: "friday"
+      time: "08:00"
+      timezone: "Europe/London"
+    labels:
+      - "dependencies"
+      - "github-actions"
+
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+      day: "friday"
+      time: "08:00"
+      timezone: "Europe/London"
+    versioning-strategy: increase-if-necessary
+    labels:
+      - "dependencies"
+      - "node"
+
+  - package-ecosystem: "pip"
```

Comment:
I'm getting the below in my forked repo. Does the `pip` ecosystem need to be unique in the config?
```
Ecosystem 'pip' has overlapping directories.
```

Reply:
We can get around this by setting `target-branch: main` for one ecosystem, allegedly, will try it now.